### ISSUE TITLE
Accessibility: Fix SVG Keyboard Focus Visibility

### ISSUE THREAD

**Issue by JeroenduC** [JeroenduC/band-of-blades-companion#82]
The Location Map nodes are keyboard focusable but lack a visible focus indicator.
